types/smokehouse.d.ts

```diff
@@ -19,6 +19,7 @@
   export type ExpectedRunnerResult = {
     lhr: ExpectedLHR,
     artifacts?: Partial<Record<keyof LH.Artifacts, any>>
+    networkRequests?: any;
```

Comment:
the big question here...what do we actually want to assert about these? :)

Reply:
From the issue, it sounds like the best thing to assert is the number of requests.

If we just did `networkRequestCount` or something, there would be no way to add a `_minChromiumVersion` for specific network request assertions.

Reply:
ah, got it. Maybe `networkRequests?: Array<unknown>` then to make it clearer but we won't have to bother typing the requests saved from `static-server`?

Or could do `networkRequests?: {length: number}` if we wanted to be explicit about what we expect from test authors right now.

Both options would be forward compatible if for some reason we wanted to move to specific object shapes later (`networkRequest?: Array<{url: string}>` or whatever).